deflect/config.h

```diff
@@ -40,16 +40,20 @@
 #ifndef DEFLECT_CONFIG_H
 #define DEFLECT_CONFIG_H
 
-#ifdef __cplusplus
-#  ifndef CXX_FINAL_OVERRIDE_SUPPORTED
+#ifdef DEFLECT_USE_CXX03
```

Comment:
Nice too. Should I add Deflect to the list of projects in Jenkins which build with CXX03?